CHANGELOG.md

```diff
@@ -11,12 +11,19 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Added
 
+- List of end-devices can now be sorted by `last_seen_at` field. Unseen devices will be shown last.
```

Comment:
This line should not be here, right ?

Reply:
Yes, @nicholaspcr mentioned that this is a `v3.23.2` addition and not `v3.24`

Reply:
I think you're confusing them. This line has been added in `v3.23.1`:
https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.23/CHANGELOG.md#added-1

But the recent change that I think @nicholaspcr has in mind has indeed been targetting `v3.23.2`:
https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.23/CHANGELOG.md#fixed

Reply:
Yes, you are right, it's the line under `# Fixed`

Reply:
Indeed this line shouldn't be in the unreleased section, just the `FIXED` addition